package.json

```diff
@@ -5,11 +5,16 @@
   "author": "PyCon Taiwan <web@pycon.tw>",
   "private": true,
   "devDependencies": {
+    "@babel/core": "^7.23.2",
+    "@babel/plugin-proposal-decorators": "^7.23.2",
+    "@babel/plugin-proposal-function-sent": "^7.23.2",
+    "@babel/plugin-proposal-throw-expressions": "^7.23.2",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-syntax-import-meta": "^7.10.4",
+    "@babel/plugin-transform-class-properties": "^7.23.2",
+    "@babel/preset-env": "^7.23.2",
     "autoprefixer": "^8.0.0",
-    "babel-core": "^6.26.0",
-    "babel-preset-env": "^1.6.1",
-    "babel-preset-stage-2": "^6.24.1",
```

Comment:
please note that babel's stage preset has been deprecated and removed from babel 7
https://babeljs.io/blog/2018/07/27/removing-babels-stage-presets